Web/starter_project_iconics/pages/profile/index.tsx

```diff
@@ -0,0 +1,35 @@
+import BlogCard from '@/components/common/blogCard'
+import { card1 } from '@/store/mockData/data'
+
+const index = () => {
+  const nums: number[] = [0, 1, 2, 3, 4]
+  return (
+    <div
+      style={{
+        display: 'flex',
+        marginTop: '10px',
+        flexDirection: 'row',
+        justifyContent: 'flex-start',
+        flexWrap: 'wrap',
+        alignContent: 'flex-end',
+      }}
+    >
+      {nums?.map((value, index) => {
+        if (index % 2 === 0)
```

Comment:
if we have the list generated like i mentioned above, we wouldn't need an if condition to check the index. we can just pass the "feature" as it comes from the list (which i think is the only difference between the two)